mindjunkies/forums/tests.py

```
@@ -0,0 +1,88 @@
+        self.client.login(username="testuser", password="testpass")
+        response = self.client.post(
+            reverse("topic_submission", kwargs={"course_slug": self.course.slug, "module_id": self.module.id}),
+            {"title": "New Topic", "content": "Topic Content"}
+        )
+        self.assertEqual(response.status_code, 302)  # Redirect on success
+        self.assertTrue(ForumTopic.objects.filter(title="New Topic").exists())
+
+    def test_comment_submission_view(self):
+        self.client.login(username="testuser", password="testpass")
+        response = self.client.post(
+            reverse("comment_submission", kwargs={"course_slug": self.course.slug, "topic_slug": self.forum_topic.slug}),
```

Ruff (0.8.2)

Line too long (121 > 119)

(E501)